gnucash/ui/gnc-plugin-page-invoice-ui.xml

```diff
@@ -40,6 +40,7 @@
       <menu name="Business" action="BusinessAction">
         <placeholder name="BusinessPlaceholderMiddle">
           <menuitem name="ToolsProcessPayment"     action="ToolsProcessPaymentAction"/>
+          <menuitem name="ToolsSaveLayout"         action="ToolsSaveLayoutAction"/>
```

Comment:
I would put this under the "Window" menu rather than the tools menu as i is window state you are manipulating here.

Reply:
As with the register, provide a menu item as well to restore the state back to the original built-in state for consistency.